Services/CO.CDP.Tenant.WebApi/appsettings.Development.json

```diff
@@ -6,6 +6,6 @@
     }
   },
   "ConnectionStrings": {
-    "TenantDatabase": "Server=localhost;Database=cdp;Username=cdp;Password=cdp;"
+    "TenantDatabase": "Server=db;Database=cdp;Username=cdp_user;Password=cdp123;"
```

Comment:
Should we block all app setting files with the exception of appsettings-template.json ?

Reply:
I'd prefer to have it working without any manual steps required (like having to copy the template). Settings like database connection will be overridden with environment variables when deployed to AWS.

Reply:
@glgoaco for example, to override the connection string, the following environment variable needs to be defined:

```
ConnectionStrings__TenantDatabase="Server=localhost;Database=MyDb;Username=MYUSERPassword=MYPASS;"
```

I actually added it to the HTTP launch profile now, so that if we run the tenant service from the IDE it connects to the DB that's run on localhost (docker).

https://github.com/cabinetoffice/GCGS-Central-Digital-Platform/blob/8d0e255321b88c2694d77ecfa8083b759d25b4e1/Services/CO.CDP.Tenant.WebApi/Properties/launchSettings.json#L20